runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/environment/ProcessManager.java

```diff
@@ -63,22 +62,21 @@ private ProcessManager() {
     this.processes = Collections.synchronizedMap(new HashMap<>());
   }
 
-  static class RunningProcess {
+  public static class RunningProcess {
     private Process process;
 
     RunningProcess(Process process) {
       this.process = process;
     }
 
     /** Checks if the underlying process is still running. */
-    void isAliveOrThrow() throws IllegalStateException {
+    public void isAliveOrThrow() throws IllegalStateException {
       if (!process.isAlive()) {
         throw new IllegalStateException("Process died with exit code " + process.exitValue());
       }
     }
 
-    @VisibleForTesting
-    Process getUnderlyingProcess() {
+    public Process getUnderlyingProcess() {
```

Comment:
This leaks the ProcessManager. Could we instead add the required methods for this here?